index.js.flow

```diff
@@ -219,66 +233,68 @@ export interface SnapshotAssertion {
 
 export interface ThrowsAssertion {
 	/**
-	 * Assert that the function returns a promise that rejects with [an error](https://www.npmjs.com/package/is-error).
-	 * If so, returns the rejection reason. You must await the result.
+	 * Assert that the function throws [an error](https://www.npmjs.com/package/is-error). If so, returns the error value.
 	 */
-	(fn: () => PromiseLike<any>, expectations?: null, message?: string): Promise<any>;
+	(fn: () => any, expectations?: null, message?: string): any;
```

Comment:
Not relevant to this PR, but we still wanted to type it to return an `Error` instead of `any`, right? Per https://github.com/avajs/ava/issues/1794

Reply:
Ah yes, I guess I should be careful not to close that issue. Ran out of time though on Sunday, barely got this PR out 😄